builders/build/templates.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Templates
+description: Tanssi includes templates to kick-start the development of an Appchain, one for a Substrate-oriented runtime and another featuring full EVM (Ethereum) support.
+---
+
+# Templates {: #templates } 
+
+## Introduction {: #introduction }
+
+ContainerChains deployed through Tanssi are essentially parachains within the Polkadot ecosystem that are capable of interacting with the relay chain and other parachains. They also contain the necessary functionalities to support the Tanssi protocol.
+
+As presented in the [Included Templates](/learn/tanssi/included-templates#baseline-appchain-template){target=_blank} from the Learn section, Tanssi already provides two templates to jumpstart the development process:
+
+- **[Baseline Appchain template](/learn/tanssi/included-templates/#baseline-appchain-template){target=_blank}** - a template that provides the basic platform to start adding custom logic in a Substrate based Appchain
+- **[Baseline EVM (Ethereum Virtual Machine) Template](/learn/tanssi/included-templates/#baseline-evm-template){target=_blank}** - a template that provides full Ethereum compatibility
+
+In this article, the required base setup and how to use the templates as a starting point to start building your Appchain are presented.
+
+## Base Setup - Integrating Into Polkadot's ecosystem {: #base-setup-integrating-into-polkadot }
+
+To integrate smoothly into the Polkadot ecosystem, the ContainerChains must implement the [Cumulus SDK](https://github.com/paritytech/cumulus){target=_blank}, which manages many aspects, including: 
+
+- **Consensus** - Cumulus adds the necessary functionality to allow the collators to produce, gossip and validate the blocks, and coordinate with the relay chain to get notified about the block's finality 
+- **[XCM](https://wiki.polkadot.network/docs/learn-xcm){target=_blank}** - handles the ingestion and dispatch of incoming downward and lateral messages, allowing a ContainerChain to communicate and interoperate with the other sibling chains within the ecosystem
+- **Runtime Upgrades** - a runtime upgrade in a ContainerChain must be informed to the relay chain to allow its validators to check on the blocks produced by the collators of the ContainerChains. Cumulus notifies the upgrade to the relay chain and waits the required amount of time (blocks) before enacting the change
+
+More information about how to configure Cumulus SDK to integrate an Appchain into the Polkadot's ecosystem can be found in the [official Cumulus template](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachain-template).
+
+## Base Setup - Supporting the Tanssi Protocol {: #base-setup-supporting-tanssi }
```

Comment:
If you are reclycling a lot of the content from other sections makes sense to add snippets here!

Also, invite the reader to learn more about how the block production mechanism work by linking to the learn section part or something similar